compute/instances_test.go

```diff
@@ -30,6 +35,202 @@ func getAnyInstanceID(t *testing.T, client *compute.ComputeClient) (string, erro
 	return "", errors.New("no machines configured")
 }
 
+func RandInt() int {
+	reseed()
+	return rand.New(rand.NewSource(time.Now().UnixNano())).Int()
```

Comment:
Given we actually _need_ cryptographically secure random number generation in the `authentication` package (ref https://github.com/joyent/triton-go/blob/5f7872646922b73632f434afa5e45fcd18aa9d5b/authentication/private_key_signer.go#L77) which is imported into the client, which is in turn imported everywhere, maybe we should put any seeding there?